lib/rubocop/cop/rspec/named_subject.rb

```diff
@@ -0,0 +1,68 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Avoid referencing `subject` explicitly in specs
```

Comment:
I haven't explicitly explained why but I've updated this comment to make it more obvious. Now it says

> Give `subject` a descriptive name if you reference it directly